### PR TITLE
Wallet base ui

### PR DESCRIPTION
**Pull request by rmisio**
https://ob1company.atlassian.net/browse/OB-61

This PR includes some initial work on the top section of the wallet, notably:
- Wallet is launchable from the wallet icon on the top menu bar
- Current wallet balance is shown (transaction count is a dummy number until the transaction section will be implemented)
- Ability to toggle between the Send and Receive money sections (the content of each section is mostly incomplete / not-functioning - it will be completed in a subsequent PR)
- BTC ticker at the top of the wallet, which shows the value of 1 BTC in your currency (if your currency is not BTC)

### CODE REVIEW

File: js/views/modals/wallet/ReceiveMoney.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  }
+
+  className() {
+    return 'receiveMoney';
+  }
+
+  render() {
+    loadTemplate('modals/wallet/receiveMoney.html', (t) => {
+      this.$el.html(t({
+        qrDataUri: qr('bitcoin:mrCtpywttZSQCpW5eyYuEZfhpmVFPL8kYg',
```
Comment (rmisio) [inline js/views/modals/wallet/ReceiveMoney.js]: yes, it's a placeholder.

**Comment by jjeffryes** [comment]
@morebrownies do you want to give this a quick look? If everything looks good to you, it's ready to merge.

**Comment by morebrownies** [comment]
Yes! I will take a look in a few.

**Comment by morebrownies** [comment]
👍  Yeah looks good to me @rmisio @jjeffryes 

The only thing I noticed that is off is the Amount input is missing the currency dropdown. I'm assuming you'll pick that up on the next PR though.

screen shot 2017-04-04 at 4 46 15 pm

**Pull request merged**